tailwind/ui/buttons/ButtonTwoLines.tsx

```diff
@@ -0,0 +1,127 @@
+import type { IconType } from "react-icons/lib"
+
+import { cn } from "@/lib/utils/cn"
+
+import { Stack } from "../../../src/components/ui/flex"
+
+import {
+  Button,
+  ButtonLink,
+  type ButtonLinkProps,
+  type ButtonProps,
+} from "./Button"
+
+type CommonProps = {
+  icon: IconType
+  iconAlignment?: "left" | "right" | "start" | "end"
+  /**
+   * Reduced choices of the button variant.
+   *
+   * This component only accepts the `solid` or `outline` variant
+   */
+  variant?: "solid" | "outline"
+  /**
+   * Reduced choices of the button size
+   *
+   * This component only accepts the `md` or `sm` sizes
+   */
+  size?: "md" | "sm"
+  mainText: string
+  helperText: string
+  /**
+   * Should the main text be below the helper text instead of ab?
+   */
+  reverseTextOrder?: boolean
+}
+
+type OmittedTypes = "variant" | "size" | "children"
+
+type ButtonTypeProps = CommonProps &
+  Omit<ButtonProps, OmittedTypes> & {
+    componentType: "button"
+  }
+
+type ButtonLinkTypeProps = CommonProps &
+  Omit<ButtonLinkProps, OmittedTypes> & {
+    componentType: "link"
+  }
+
+type ButtonTwoLinesProps = ButtonTypeProps | ButtonLinkTypeProps
+
+const ButtonTwoLines = ({
```

Comment:
One minor observation, in the prod version, we have a bigger `py` when `size=md`.

`size === "md" ? "py-4" : "py-2"`

Reply:
Ahh yes, I missed migrating this over!